Code Review Assignment_JrSDET .java

```diff
@@ -55,7 +55,7 @@ public static void main(String[] args) throws InterruptedException, AWTException
 		WebElement addCreate = driver.findElement(By.xpath("//button[@class ='hf-entity-footer_primary hf-primary-action ember-view']"));
 	      addCreate.click();
 
-	      Thread.sleep(3000);
+	      Thread.sleep(3000); // code review

```

Comment:
Thread.sleep() is used too many times which results in flaky code. To avoid this we can make use of explicit wait (i.e WebDriverWait) when we need to wait specific amount for a webElement and implicit wait when we will be waiting uniform amount of time for most of the methods. This way it waits according to the state of website and not some hardcoded static wait.